scripts/ci/install-go-deps.sh

```diff
@@ -16,7 +16,7 @@ go get -f -u github.com/tebeka/go2xunit
 export PATH=$PATH:$GOPATH/bin
 
 # speedup govendor sync command
-curl -o /tmp/vendor.tgz http://46.231.132.68:8080/v1/AUTH_0ec9e4f4f3044236b4d18536ccfcb182/skydive/vendor/vendor.tar.gz
+curl -o /tmp/vendor.tgz curl -o /tmp/vendor.tgz http://ci.skydive.network/vendor.tar.gz
```

Comment:
look like some copy paste error? curl -o /tmp/vendor.tgz is repeating two times.

Reply:
yep that it